org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -493,6 +493,91 @@ package lang {
 	}
 	
 	/**
+	 * Represents a set of key -> values
+	 * 
+	 */
+	class Dictionary {
+	
+		constructor() { }
+		/**
+		 * Adds or updates a value based on a key
+		 */
+		method put(_key, _value) native
+		
+		/*
+		 * Returns the value to which the specified key is mapped, or null if this Dictionary contains no mapping for the key.
```

Comment:
Nevertheless, the underlying java HashMap allows for null values, so there might be inconsistencies.